config/config.go

```diff
@@ -30,32 +32,37 @@ func Init() {
 		log.Fatal(err)
 	}
 
-	configFile := filepath.Join(configDir, "ght", "config.yaml")
-
-	b, err := ioutil.ReadFile(configFile)
-	if err != nil {
-		log.Fatal(err)
-	}
 	logFile := filepath.Join(configDir, "ght", "debug.log")
 	output, err := os.Create(logFile)
 	if err != nil {
 		log.Fatal(err)
 	}
-	log.SetOutput(output)
+
+	log.SetOutput(io.MultiWriter(output, os.Stderr))
+
+	configFile := filepath.Join(configDir, "ght", "config.yaml")
+
+	b, err := os.ReadFile(configFile)
+	if err != nil {
+		if !os.IsNotExist(err) {
+			log.Fatal(err)
+		}
+
+		log.Fatalf("Could not find configuration file, %s", readThisMessage)
+	}
 
 	var conf struct {
 		GitHub github `yaml:"github"`
 	}
 
 	if err := yaml.Unmarshal(b, &conf); err != nil {
-		log.Fatal(err)
+		log.Fatalf("cannot deserialize config file: %s", err.Error())
```

Comment:
I think we do not need call `Error()` method
```suggestion
		log.Fatalf("cannot deserialize config file: %s", err)
```